skimage/viewer/tests/test_utils.py

```diff
@@ -29,7 +32,7 @@ def test_open_file_dialog():
     assert filename is None
 
 
-@testing.skipif(not has_qt, reason="Qt not installed")
+@pytest.mark.skipif(not has_qt, reason="Qt not installed")
```

Comment:
It looks like instances of "Qt not installed" are more numerous than "Needs matplotlib" so maybe the consistency concern should be the other way around.

Reply:
Thank @mkcor for your review, I fixed it the way around: "Needs matplotlib" -> "matplotlib not installed"